tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -134,59 +134,79 @@ public function testFixProvider()
                 '<?php function xyz(
                     $a=10,      //comment1
                     $b=20,      //comment2
-                    $c=30) {
+                    $c=30
+                ) {
                 }',
             ],
             'must keep align comments (2)' => [
                 '<?php function xyz(
                     $a=10,  //comment1
                     $b=2000,//comment2
-                    $c=30) {
-                }',
-            ],
-            'multiline comments also must be ignored (I)' => [
-                '<?php function xyz(
-                    $a=10,  /* comment1a
-                               comment1b
-                            */
-                    $b=2000,/* comment2a
-                        comment 2b
-                        comment 2c */
-                    $c=30) {
+                    $c=30
+                ) {
                 }',
-            ],
-            'multiline comments also must be ignored (II)' => [
-                '<?php
-                    function xyz(
-                        $a=10, /* multiline comment
-                                 not at the end of line
-                                */ $b=2000,
-                        $a2=10 /* multiline comment
-                                 not at the end of line
-                                */ , $b2=2000,
-                        $c=30) {
-                    }',
+            ),
+            'multiline comments also must be ignored' => array(
+                <<<'EXPECTED'
+<?php function xyz(
```

Comment:
:-1: for all that changes.
having code indented actually have a reason - we were trapped few times that fixer works when code was not extra intended, but crashed when it was.

like this works:
```php
if (1) {
    a();
}
```

but this not:
```php
    if (1) {
        a();
    }
````

you could achieve proper argument separation without changing indentation